lib/configfiles/rhel7.xml

```diff
@@ -2,6 +2,38 @@
 <froxlor>
 	<distribution name="RHEL" version="7"
 		defaulteditor="/usr/bin/nano">
+      <defaults>
```

Comment:
What about defaults for the other distros? :)

Reply:
i've only set defaults for RHEL systems as those are the onces i know for certain.
if someone provides defaults for other systems those will be taken into account.